Project-I-Social-Card/social-card/src/index.js

```diff
@@ -4,3 +4,19 @@ import './index.css';
 import App from './App';
 
 ReactDOM.render(<App />, document.getElementById('root'));
+
+// interactivity for footer icons
+const icons = document.querySelectorAll('.footer-container i');
```

Comment:
As I told you all this morning - you want to keep the logic as close as possible to the component. Here it would make more sense to put this logic in <Footer />